test/factories/accounts/user_factory.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Cadet.Accounts.UserFactory do
+  @moduledoc """
+  Factory(ies) for Cadet.Accounts.User entity
+  """
+
+  defmacro __using__(_opts) do
+    quote do
+      alias Cadet.Accounts.User
+
+      def user_factory do
+        %User{
+          name: Faker.Name.En.name(),
+          role: :staff
```

Comment:
Let's set this to random role? although it doesn't really make a difference since we usually specify a role when creating the user anyway.

Reply:
Done.